.mandatory_files

```diff
@@ -19,9 +19,8 @@ Dockerfile
 config_schema.json
 example_config.yaml
 LICENSE
+pyproject.toml
 README.md
-setup.py
-setup.cfg
```

Comment:
setup.py and setup.cfg should also be added to the .deprecated_files list so that they are automatically removed